js/core/component_registrator.d.ts

```diff
@@ -7,7 +15,7 @@
  * @namespace DevExpress
  * @hidden
  */
-export default function registerComponent(name: string, componentClass: any): void;
+export default function registerComponent<TComponent>(name: string, componentClass: IComponentFactory<TComponent>): void;
```

Comment:
@IlyaKhD Making several default exports in a single file - is it a "legal" scenario?